frontends/concrete-python/tests/execution/test_min_max.py

```diff
@@ -11,279 +11,141 @@
-        (
-            "maximum_optimized_y",
-            lambda x, y: np.maximum(x, fhe.hint(y, bit_width=4)),  # type: ignore
-        ),
-        # bit widths
-        4,
-        3,
-        # signednesses
-        False,
-        True,
-        # shapes
-        (),
-        (),
-        # strategy
-        fhe.MinMaxStrategy.CHUNKED,
-    ],
-]
-cases += [
-    [
-        # operation
-        operation,
-        # bit widths
-        1,
-        1,
-        # signednesses
-        lhs_is_signed,
-        rhs_is_signed,
-        # shapes
-        (),
-        (),
-        # strategy
-        fhe.MinMaxStrategy.CHUNKED,
-    ]
-    for lhs_is_signed in [False, True]
-    for rhs_is_signed in [False, True]
-    for operation in [
-        (
-            "maximum",
-            lambda x, y: np.maximum(x, y),
-        ),
-    ]
-]
-cases = [
-    [
-        # operation
-        ("maximum_increased_bit_widths", lambda x, y: (np.maximum(x, y), x + 100, y + 100)),
-        # bit widths
-        7,
-        7,
-        # signednesses
-        True,
-        False,
-        # shapes
-        (),
-        (),
-        # strategy
-        fhe.MinMaxStrategy.CHUNKED,
-    ],
-    [
-        # operation
-        ("maximum_increased_bit_widths", lambda x, y: (np.maximum(x, y), x + 100, y + 100)),
-        # bit widths
-        7,
-        7,
-        # signednesses
-        False,
-        True,
-        # shapes
-        (),
-        (),
-        # strategy
-        fhe.MinMaxStrategy.CHUNKED,
-    ],
-]
-for lhs_bit_width in range(1, 5):
-    for rhs_bit_width in range(1, 5):
-        strategies = []
-        if lhs_bit_width <= 3 and rhs_bit_width <= 3:
-            strategies += [
-                fhe.MinMaxStrategy.ONE_TLU_PROMOTED,
-                fhe.MinMaxStrategy.THREE_TLU_CASTED,
-            ]
-        else:
-            strategies += [
-                fhe.MinMaxStrategy.CHUNKED,
-            ]
-
-        for lhs_is_signed in [False, True]:
-            for rhs_is_signed in [False, True]:
-                cases += [
-                    [
-                        # operation
-                        operation,
-                        # bit widths
-                        lhs_bit_width,
-                        rhs_bit_width,
-                        # signednesses
-                        lhs_is_signed,
-                        rhs_is_signed,
-                        # shapes
-                        random.choice([(), (2,), (3, 2)]),
-                        random.choice([(), (2,), (3, 2)]),
-                        # strategy
-                        strategy,
-                    ]
-                    for operation in [
-                        ("minimum", lambda x, y: np.minimum(x, y)),
-                        ("maximum", lambda x, y: np.maximum(x, y)),
-                    ]
-                    for strategy in strategies
-                ]
+cases = []
+for operation in [("max", lambda x: np.max(x)), ("min", lambda x: np.min(x))]:
```

Comment:
If I am correct, the fact that you don't test cases where the output shape is different from input shape prevent most of your implementation from being tested.

Reply:
Input shape depend on output shape and axis argument. Specified axes get's destroyed, so I think it's tested.

Reply:
But of course, coverage report will tell the truth, so let's see.

Reply:
Well at least I wasn't able to trigger a breakpoint in this section with the tests, but let's see yes !